.env

```
@@ -1,3 +1,3 @@
-VITE_APP_RAPID_API_KEY=ffbb530b15msh0f64b40ee714ba8p1cb5d8jsnedbd8f738085
+VITE_APP_RAPID_API_KEY=7b6ff654eamsh9c748f36d11dbc3p140376jsn9bc92ef7d2b2
 VITE_APP_RAPID_API_HOST=youtube138.p.rapidapi.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_APP_RAPID_API_HOST key should go before the VITE_APP_RAPID_API_KEY key

(UnorderedKey)